user/build-environment-updates/2017-10-26.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Build Environment Update History
+layout: en
+permalink: /user/build-environment-updates/2017-10-26/
+category: build_env_updates
+date: 2017-10-26
+---
+
+## 2017-10-26
+
+This update applies to macos10.12-xcode8.3 image, for `os: osx` builds
+
+### Schedule
+
+[2017-10-26 14:00 UTC](http://everytimezone.com/#2017-10-26,240,c8l)
+
+### Added
+
+
+
+### Changed
+
+- Bundler version `1.15.4`
+- xctool version `0.3.3`
+- Darwin Kernel Version `16.7.0`
+- automake `1.15.1`
+- boost `1.65.1`
+- carthage `0.26.0`
+- cgal `4.11`
+- cmake `3.9.4`
+- coreutils `8.28_1`
+- freexl `1.0.4`
+- gdal `1.11.5_3`
+- geos `3.6.2`
+- git `2.14.2`
+- gnupg `2.2.1`
+- gnutls `3.5.15`
+- go `1.9.1`
+- icu4c `59.1`
+- jpeg `9b`
+- json-c `0.12.1`
+- libgcrypt `1.8.1`
+- libgeotiff `1.4.2_1`
+- libpng `1.6.34`
+- libtiff `4.0.8_4`
+- libxml2 `2.9.6`
+- lzlib `1.9`
+- mercurial `4.3.3`
+- mpfr `3.1.6`
+- msgpack `2.1.5`
+- p11-kit `0.23.9`
+- node `8.6.0`
+- motion version `5.3`
+- pod version `1.3.1`
+- postgis `2.4.0`
+- postgresql `9.6.5`
+- pyenv `1.1.5`
+- python `2.7.14`
+- swiftlint `0.23.1`
+
+
+### Deprecated
+
+N/A
+
+
+### Removed
+
+N/A
+
+### Fixed
+
+N/A
+
+### Security
+
+- All brew packages updated to include latest security patches.
+
+### Details
+
+- [xcode8.3 diff](https://github.com/Lyoness/travis_staging_test/commit/fba03279686a1492a78ed958d72344fe4af3a55f?diff=unified#diff-934df722cbc32c134935cba6c22c3a84R740)
```

Comment:
linking to a diff here for those that prefer the full details. Linux images have https://stackmeta-production.herokuapp.com/  to produce this, for future consideration